### DISCUSSION TITLE
sudo tail -f /var/log/fail2ban.log help please

### DISCUSSION THREAD

**Discussion by carrabelloy** [fail2ban/fail2ban#3484] [Q&A] [answered]
sudo tail -f /var/log/fail2ban.log
    -x                      force execution of the server (remove socket file)
    -b                      start server in background (default)
    -f                      start server in foreground
    --async                 start server in async mode (for internal usage only, don't read configuration)
    --timeout               timeout to wait for the server (for internal usage only, don't read configuration)
    --str2sec <STRING>      convert time abbreviation format to seconds
    -h, --help              display this help message
    -V, --version           print the version (-V returns machine-readable short format)

Report bugs to https://github.com/fail2ban/fail2ban/issues
sudo fail2ban-server -o -x -f start

**Comment by sebres** [comment, marked as answer]
What is it exactly?
Is it start of fail2ban-server from systemd unit? (I don't understand what and how it is logging to fail2ban.log)
How it was installed? Which version, etc?

Regarding systemd unit, take a look in our unit template, the command is https://github.com/fail2ban/fail2ban/blob/a9b30eb86ea4367e7464c90f517b1e1da9c88020/files/fail2ban.service.in#L11

> sudo fail2ban-server -o -x -f start

What do you try to do? To start the server in foreground the command must be like above. To start server in background it must be without option `-f` (and normally one uses `fail2ban-client` instead). 
And option `-o` is unknown here (and was never available in no one version).

**Reply by carrabelloy** [reply]
Make sure that the file contains only one [Definition] section and that there are no duplicates or extra characters or spaces.

If you still have trouble, you can also reset the file to the default configuration. You can copy the default configuration from here and replace the existing sshd.conf file. After replacing the file, restart Fail2Ban and check the status:
Make sure the file contains only one [definition] section. Remove any additional [definition] sections or other duplicates. The file should look like the following:

[INCLUDES]

before = common.conf

[definition]

_daemon = sshd

failregex = ^%(__prefix_line)s(?:error: PAM: )?Authentication failure for .* from \s*$
^%(__prefix_line)s(?:error: PAM: )?User not known to the underlying authentication module for .* from \s*$
^%(__prefix_line)sFailed (?:password|publickey) for .* from (?: port \d*)?(?: ssh\d*)?$
^%(__prefix_line)sFailed (?:password|publickey) for invalid user .* from (?: port \d*)?(?: ssh\d*)?$
^%(__prefix_line)sFailed (?:password|publickey) for .* from port \d* ssh\d*$
^%(__prefix_line)s(?:error: PAM: )?User not known to the underlying authentication module for .* from \s*$
^%(__prefix_line)sConnection from port \d+(?: on \S+ port \d+)?$
^%(__prefix_line)sReceived disconnect from : 3: \S+\s*$
^%(__prefix_line)sUser .+ from not allowed because not listed in AllowUsers\s*$
^%(__prefix_line)sUser .+ from not allowed because listed in DenyUsers\s*$
^%(__prefix_line)sUser .+ from not allowed because not in any group\s*$
^%(__prefix_line)sUser .+ from not allowed because a group is listed in DenyGroups\s*$
^%(__prefix_line)sUser .+ from not allowed because none of user's groups are listed in AllowGroups\s*$
^%(__prefix_line)srefused connect from \S+ ()\s*$
^%(__prefix_line)sAddress .* POSSIBLE BREAK-IN ATTEMPT!\s*$

ignoreregex =
As errors have continued to appear, the decision has been made to reset the whole thing And this is that way I have now been shown the way on I am sorry that the problem still exists. T[here](https://github.com/fail2ban/fail2ban/blob/0.11/config/filter.d/sshd.conf) seems to be still a problem in the sshd.conf file. I will ask you to check the contents of the file again and make sure there are no duplicates. sudo nano /etc/fail2ban/filter.d/sshd.conf
Make sure that the file contains only a [definition] section and that there are no duplicates or extra characters or spaces.
If you still have trouble, you can also reset the file to the default configuration. You can copy the default configuration from here and replace the existing sshd.conf file. After replacing the file, restart Fail2Ban and check the status:
https://github.com/fail2ban/fail2ban/blob/0.11/config/filter.d/sshd.conf
